### PR TITLE
refactor nmstate to follow standard folder structure

### PR DESCRIPTION
**Pull request by strangiato**

### CODE REVIEW

File: nmstate/README.md [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
+Do not use the `base` directory directly, as you will need to patch the `channel` based on the version of OpenShift you are using, or the version of the operator you want to use.
+
+The current *overlays* available are for the following channels:
+
+* [stable](operator/overlays/stable)
+
+## Usage
 
-Install the NMState operator:
+If you have cloned the `gitops-catalog` repository, you can install NVIDIA GPU Operator based on the overlay of your choice by running from the root (`gitops-catalog`) directory.
```
Comment (dlbewley) [inline nmstate/README.md:13]: Oops. `s/NVIDIA GPU/NMState/`

File: nmstate/README.md [hunk 2]
```diff
@@ -1,25 +1,32 @@
 # NMState
 
-## Usage
-
 This is to deploy [NMState operator](https://docs.openshift.com/container-platform/4.9/networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.html) to OpenShift. The NMState Operator provides users with functionality to configure various network interface types, DNS, and routing on cluster nodes
```
Comment (dlbewley) [inline nmstate/README.md:3]: Possibly update to 4.14 docs? Or 'latest'?
https://docs.openshift.com/container-platform/latest/networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.html

File: nmstate/README.md (hunk 1 above)
Comment (strangiato) [reply nmstate/README.md]: Woops.  Good catch!

File: nmstate/README.md (hunk 2 above)
Comment (strangiato) [reply nmstate/README.md]: Great idea!  Done.

**Review by pittar — APPROVED** [review]
LGTM!

**Pull request merged**